File: 프로그래머스/lv0/120891. 369게임/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0, sum = 0;
        String st = Integer.toString(order);
        
        for(int i = 0; i<st.length(); i++){
            if((st.charAt(i) - 48)%3 == 0){
                if((st.charAt(i)-48) == 0) continue;
                sum++;
            }
            System.out.println(st.charAt(i)-48);
        }
        return answer = sum;
    }
}

File: 프로그래머스/0/120891. 369게임/369게임.js
function solution(order) {
    var answer = 0;
    var str = String(order);
    
    for(let i = 0; i<str.length; i++){
        if(Number(str[i])%3 == 0 && str[i] != '0') answer++;
    }
    return answer;
}